src/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Application.Queries/GetWorkplaceTranslationsQuery.g.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Generated
+
+#nullable enable
+
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+using Nox.Application.Commands;
+using YamlDotNet.Core.Tokens;
+using ClientApi.Application.Dto;
+using ClientApi.Infrastructure.Persistence;
+
+namespace ClientApi.Application.Queries;
+
+public record  GetWorkplaceTranslationsQuery(System.Int64 keyId) : IRequest <IQueryable<WorkplaceLocalizedDto>>;
```

Comment:
To allow extend, although I'm not seeing a useful scenario at the moment
```suggestion
public partial record  GetWorkplaceTranslationsQuery(System.Int64 keyId) : IRequest <IQueryable<WorkplaceLocalizedDto>>;
```